config/tags.config

```diff
@@ -23,32 +23,32 @@
 # For hotspot version 10.23 / alpine / nightly /slim
 # alpine-nightly-slim 10.23-alpine-nightly-slim x86_64-alpine-10.23-nightly-slim
 
-ubuntu-jdk-releases-full-tags: latest {{ JDK_RELEASES_VER }} {{ ARCH }}-ubuntu-{{ JDK_RELEASES_VER }}
-ubuntu-jdk-releases-slim-tags: slim {{ JDK_RELEASES_VER }}-slim {{ ARCH }}-ubuntu-{{ JDK_RELEASES_VER }}-slim
+ubuntu-jdk-releases-full-tags: latest ubuntu {{ JDK_RELEASES_VER }}-ubuntu {{ ARCH }}-ubuntu-{{ JDK_RELEASES_VER }}
+ubuntu-jdk-releases-slim-tags: slim {{ JDK_RELEASES_VER }}-ubuntu-slim {{ ARCH }}-ubuntu-{{ JDK_RELEASES_VER }}-slim
 ubuntu-jdk-nightly-full-tags: nightly {{ JDK_NIGHTLY_VER }}-nightly {{ ARCH }}-ubuntu-{{ JDK_NIGHTLY_VER }}-nightly
 ubuntu-jdk-nightly-slim-tags: nightly-slim {{ JDK_NIGHTLY_VER }}-nightly-slim {{ ARCH }}-ubuntu-{{ JDK_NIGHTLY_VER }}-nightly-slim
 
-debian-jdk-releases-full-tags: debian {{ JDK_RELEASES_VER }} {{ ARCH }}-debian-{{ JDK_RELEASES_VER }}
+debian-jdk-releases-full-tags: debian {{ JDK_RELEASES_VER }}-debian {{ ARCH }}-debian-{{ JDK_RELEASES_VER }}
 debian-jdk-releases-slim-tags: debian-slim {{ JDK_RELEASES_VER }}-debian-slim {{ ARCH }}-debian-{{ JDK_RELEASES_VER }}-slim
 debian-jdk-nightly-full-tags: debian-nightly {{ JDK_NIGHTLY_VER }}-debian-nightly {{ ARCH }}-debian-{{ JDK_NIGHTLY_VER }}-nightly
 debian-jdk-nightly-slim-tags: debian-nightly-slim {{ JDK_NIGHTLY_VER }}-debian-nightly-slim {{ ARCH }}-debian-{{ JDK_NIGHTLY_VER }}-nightly-slim
 
-windows-jdk-releases-full-tags: windows {{ JDK_RELEASES_VER }} {{ ARCH }}-windows-{{ JDK_RELEASES_VER }}
+windows-jdk-releases-full-tags: windows {{ JDK_RELEASES_VER }}-windows {{ ARCH }}-windows-{{ JDK_RELEASES_VER }}
 windows-jdk-nightly-full-tags: windows-nightly {{ JDK_NIGHTLY_VER }}-windows-nightly {{ ARCH }}-windows-{{ JDK_NIGHTLY_VER }}-nightly
 
 alpine-jdk-releases-full-tags: alpine {{ JDK_RELEASES_VER }}-alpine {{ ARCH }}-alpine-{{ JDK_RELEASES_VER }}
 alpine-jdk-releases-slim-tags: alpine-slim {{ JDK_RELEASES_VER }}-alpine-slim {{ ARCH }}-alpine-{{ JDK_RELEASES_VER }}-slim
 alpine-jdk-nightly-full-tags: alpine-nightly {{ JDK_NIGHTLY_VER }}-alpine-nightly {{ ARCH }}-alpine-{{ JDK_NIGHTLY_VER }}-nightly
 alpine-jdk-nightly-slim-tags: alpine-nightly-slim {{ JDK_NIGHTLY_VER }}-alpine-nightly-slim {{ ARCH }}-alpine-{{ JDK_NIGHTLY_VER }}-nightly-slim
 
-ubuntu-jre-releases-full-tags: jre {{ JDK_RELEASES_VER }} {{ ARCH }}-ubuntu-{{ JDK_RELEASES_VER }}
-ubuntu-jre-nightly-full-tags: jre-nightly {{ JDK_NIGHTLY_VER }}-nightly {{ ARCH }}-ubuntu-{{ JDK_NIGHTLY_VER }}-nightly
+ubuntu-jre-releases-full-tags: jre {{ JDK_RELEASES_VER }}-jre {{ ARCH }}-ubuntu-{{ JDK_RELEASES_VER }}-jre
```

Comment:
Since we are adding OS tags for all, I think this should be `ubuntu-jre` .

Reply:
done @dinogun